### DISCUSSION TITLE
Support other major mediaservers

### DISCUSSION THREAD

**Discussion by zidesm** [wizarrrr/wizarr#17] [Feature Requests]
Do you plan on supporting a few of the other mediaservers like emby, jellyfin?

**Comment by mtthidoteu** [comment]
Yes, Jellyfin support is planned!

**Comment by SC8198** [comment]
Is this still the case? saw it was pulled from the readme

**Comment by bleak183** [comment]
Jellyfin support has been added since long ago (as well as Emby). No plans for future platforms so I'll be closing this.
